packages/ai-chat/src/types/instance/ChatInstance.ts

```diff
@@ -47,13 +47,32 @@ export interface ChatInstance extends EventHandlers, ChatActions {
 }
 
 /**
- * This is the state made available by calling getState. This is a public method that returns immutable values.
+ * This is the state made available by calling {@link ChatInstance.getState}. This is a public method that returns immutable values.
  *
  * @category Instance
  */
 export type PublicChatState = Readonly<
   Omit<PersistedState, "humanAgentState"> & {
+    /**
+     * Current human agent state.
+     */
     humanAgent: PublicChatHumanAgentState;
+
+    /**
+     * Counter that indicates if a message is loading and a loading indicator should be displayed.
+     * If "0" then we do not show loading indicator.
+     */
+    isMessageLoadingCounter: number;
+
+    /**
+     * Optional string to display next to the loading indicator.
+     */
+    isMessageLoadingMessage?: string;
```

Comment:
Finding the `isMessageLoadingMessage` name a bit confusing, could we rename to something like `isMessageLoadingText` or `loadingIndicatorText`?